underfs/hdfs/src/main/java/alluxio/underfs/hdfs/AlluxioHdfsException.java

```diff
@@ -30,23 +30,28 @@
  * org/apache/hadoop/hdfs/web/resources/ExceptionHandler.java
  */
 public class AlluxioHdfsException extends AlluxioRuntimeException {
+  private static final ErrorType ERROR_TYPE = ErrorType.External;
```

Comment:
we may want to set errorType based on exception got from hdfs @rongrong what's your thought